pkg/rhcos/stream_fcos.go

```diff
@@ -5,3 +5,9 @@ package rhcos
 func getStreamFileName() string {
 	return "coreos/fcos.json"
 }
+
+func getMarketplaceStreamFileName() string {
```

Comment:
Is this name not causing issues? There are 3 defined in the same package. They may not be exported but they are all in the same package.